map/autoware_dynamic_lanelet_provider/config/dynamic_lanelet_provider.param.yaml

```diff
@@ -0,0 +1,5 @@
+/**:
+  ros__parameters:
+    dynamic_map_loading_grid_size: 50.0         # [m]
+    dynamic_map_loading_update_distance: 100.0  # [m]
+    dynamic_map_loading_map_radius: 200.0       # [m]
```

Comment:
@maxime-clem @takayuki5168 Could you help determine if these parameters are appropriate?

Reply:
@maxime-clem @takayuki5168 please answer :bow:

Reply:
With the current speed limit of Autoware (15km/h)  these parameters are appropriate yes (it takes 24 seconds to travel 100m at 15km/h).
In the future if we increase the speed limit we may need to adjust these parameters.
By the way, the actual values used are the ones here (currently set to the same values) : https://github.com/autowarefoundation/autoware_launch/pull/872

Reply:
@maxime-clem OK! Thank you for the answer!